### ISSUE TITLE
AVT 3 React Inline Loading Component - VoiceOver does not announce loaded state

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2896]
## Environment
Tested on macOS Mojave 10.14.5, Version 74.0.3729.157 (Official Build) (64-bit), with VoiceOver (VO)

## Description

1. When page first loads VO announces "Active Loading Indicator" as expected.
2. Ctrl-Option-Right Arrow VO announces "Active Loading Indicator group" and then "Loading data" as expected.
3. After the page loads VO fails to announce that the page has loaded.
4. This issue is also present in the UX Example.

There are actually 2 issues in the example. When inlineLoading is a success:
1. The text should change to "Data Loaded" (the UX Example it should change to "Submitted").  However, the documentation states the inlineLoading component should never be used for loading data. My recommendation is to have the example match the usage documentation.
2.  "Data Loaded" (or UX Example "Submitted") must be announced so a screen reader user knows the state has changed. This can be accomplished by adding aria-live="assertive" when onSuccess. Also, note that while the Usage documentation states descriptive text is optional there still needs to be an aria-label or aria-labelledby for each state (submitting/submitted) so a screen reader user understands what is going on.
